firestore/query.go

```diff
@@ -500,8 +500,12 @@ func (f filter) toProto() (*pb.StructuredQuery_Filter, error) {
 		op = pb.StructuredQuery_FieldFilter_GREATER_THAN_OR_EQUAL
 	case "==":
 		op = pb.StructuredQuery_FieldFilter_EQUAL
+	case "!=":
+		op = pb.StructuredQuery_Filter_NOT_EQUAL
```

Comment:
Hi, I have a comment if you don't mind.
kokoro failed with 

> ./query.go:504:8: undefined: "google.golang.org/genproto/googleapis/firestore/v1".StructuredQuery_Filter_NOT_EQUAL

It seems like you need to use `pb.StructuredQuery_FieldFilter_NOT_EQUAL` here.

Reply:
Thanks for pointing this out! I actually thought I fixed this issue last week (because I was using my fork for one of my projects and got the same error), but apparently I lost the fix again when rebasing for the intergration tests - fail!

Anyway, should be fixed now. :-)